lib/resValidator.js

```diff
@@ -68,9 +68,14 @@ module.exports = function(pathToOpenApiSpec) {
 
   function parseOpenApiFile(filePath) {
     const fileData = fs.readFileSync(filePath);
-    const openApiSpec = yaml.safeLoad(fileData);
+    let openApiSpec;
+    try {
+      openApiSpec = yaml.safeLoad(fileData);
+    } catch (error) {
+      throw new Error('Error: Unable to read the specified OpenApi document. Possible cause: file is invalid yml or JSON');
+    }
     if (!openApiSpec) {
-      throw new Error('Error: Unable to read the specified OpenApi document. Possible cause: file isn\'t .yml');
+      throw new Error('Error: Unable to read the specified OpenApi document. Possible cause: file is neither yml nor JSON');
```

Comment:
Perhaps we should just make it something like: 
```suggestion
      throw new Error('Error: Unable to read the specified OpenApi document. Supported file types: [.yml, .json]');
```